examples/js/e-commerce-umd/src/widgets/Brands.ts

```diff
@@ -3,7 +3,9 @@ import { collapseButtonText } from '../templates/panel';
 const { panel, refinementList } = window.instantsearch.widgets;
 const brandRefinementList = panel({
```

Comment:
```suggestion
const brandRefinementList = panel<typeof refinementList>({
```

I think it needs something similar to this for the types to work? not sure though